main: added lines to take position from the player

Amitava123 opened the pull request:
Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Mentioned the issue number that it fixes
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

Amitava123 committed:
Merge pull request #2 from BytesClub/master

closing issue #3

Amitava123 committed:
modifying line 32

adding codes to take input from the user

fixes: #3

signed off by Amitava Mitra <mitraamitava7@gmail.com>

Amitava123 committed:
line no 33

adding codes for the user to get the position of X or O.

fixes: #2

signed off by: Amitava Mirea <mitraamitava7@gmail.com>

0xTheProDev commented:
Please provide a clear commit message with your **Pull Request**.
Format is specified in README.

Amitava123 commented:
ok changing my commit message

Amitava123 committed:
shortlog: adding lines in main.c at line 32

adding codes so that players can input 'X' or 'O' in the game.

fixes: #2

signed-off by: Amitava123<mitraamitava7@gmail.com>
